src/advanced_inv.cpp

```diff
@@ -259,7 +266,12 @@ void advanced_inventory::print_items( advanced_inventory_pane &pane, bool active
             } else {
                 maxvolume = g->m.max_volume( s.pos );
             }
-            head = string_format( "%3.1f %3d/%3d", convert_weight( s.weight ), s.volume / units::legacy_volume_factor, maxvolume / units::legacy_volume_factor );
+            head = string_format( "%3.1f %s  %.1f/%-.1f %s",
```

Comment:
I am sorry, I do not understand this one.
Are you talking about the precision of the weight ?